.travis.yml

```diff
@@ -1,10 +1,10 @@
-jdk:
-  - oraclejdk8 # openJDK crashes sometimes; tensorframes is compiled w.r.t JDK8
-
 sudo: required
 
 dist: trusty
 
+jdk:
```

Comment:
This is pretty frustrating. There is a separate PR for this, https://github.com/databricks/spark-deep-learning/pull/20. Adding travis related stuff to your PR not only introduces merge conflicts, which then take time to resolve. It also makes your PR harder to review and we end up having the same conversations twice.

Can we commit to these conditions for merging this PR. 1) Fix failing tests in master 2) Merge travis PR 3) rebase on master, then we can properly merge this PR.